Content.Shared/Gibbing/Systems/GibbingSystem.cs

```diff
@@ -187,7 +187,7 @@ public sealed class GibbingSystem : EntitySystem
             _audioSystem.PlayPredicted(gibbable.Comp.GibSound, parentXform.Coordinates, null);
         }
 
-        if (gibType == GibType.Gib)
+        if (gibType == GibType.Gib && IsClientSide(gibbable))
             QueueDel(gibbable);
         return true;
```

Comment:
IsClientSide is never true on server

Reply:
Should I be changing this to !IsClient or removing it?